extensions/wikia/PhalanxII/models/PhalanxModel.php

```diff
@@ -103,12 +103,12 @@ public function isOk() {
 		);
 	}
 
-	public function __call($name, $args) {
-		$method = substr($name, 0, 3);
+	public function __call( $name, $args ) {
+		$method = substr( $name, 0, 3 );
 		$key = lcfirst( substr( $name, 3 ) );
 
 		$result = null;
-		switch($method) {
+		switch( $method ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this "switch" statement with "if" statements to increase readability. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1301)